winbuild/Makefile.vc

```diff
@@ -21,10 +22,20 @@ CFGSET=true
 !MESSAGE                                  Libraries can be fetched at http://pecl2.php.net/downloads/php-windows-builds/
 !MESSAGE                                  Uncompress them into the deps folder.
 !MESSAGE   WITH_SSL=<dll or static>     - Enable OpenSSL support, DLL or static
+!MESSAGE      SSL_LIBS=<File1 File2>    - OpenSSL libs to use (default: libeay32.lib ssleay32.lib) (requires WITH_SSL)
+!MESSAGE      SSL_HEADER_PATH=<path>    - Path to OpenSSL include files. (requires WITH_SSL)
+!MESSAGE      SSL_CFLAGS=...            - Extra flags to supply when compiling OpenSSL modules (requires WITH_SSL)
```

Comment:
Again, I'm following the lead in the slave file.  But in this case keeping the lib and cflags separate for each downstream project saves inadvertent header or lib pollution.  I know its unlikely but I don't want an zlib.h which is on the openSSL path to get in the way of the zlib.h I specify via `ZLIB_CFLAGS="/I%ROOT_DIR%%ZLIB_DIR%"`